crates/control/src/game_state_filter.rs

```diff
@@ -264,8 +281,15 @@ impl State {
                     ball_is_free: !opponent_kick_off && !opponent_sub_state,
                 }
             }
-            State::Playing => FilteredGameState::Playing {
-                ball_is_free: !(is_in_sub_state && opponent_is_kicking_team),
+            State::Playing => match game_controller_state.sub_state {
+                Some(SubState::PenaltyKick) => FilteredGameState::Playing {
+                    ball_is_free: !(is_in_sub_state && opponent_is_kicking_team),
+                },
+                _ => FilteredGameState::Playing {
+                    ball_is_free: !(is_in_sub_state
```

Comment:
> This also applies to the `WhistleInSet` above

I did not touch 'WhistleInSet' in this pull request, so I am not sure if you want me to clean up 'WhistleInSet" or merge 'WhistleInSet' and 'Playing' in order to avoid code duplication